### PR TITLE
Add timestamp to Markdown and JSON outputs

### PR DESCRIPTION
**Pull request by samgoldman**
Added a timestamp to the generated markdown and json files so that anyone who uses them (and might have copied them) knows how up to date they are.

### CODE REVIEW

File: tools/data_builder.py [hunk 1]
```diff
@@ -177,7 +181,8 @@ def to_json_file(data, target_path):
     data_with_meta = {
         "edit_at": "https://github.com/2020PB/police-brutality",
         "help": "ask @ubershmekel on twitter",
-        "data": data,
+        "timestamp": str(datetime.now()),
```
Comment (ubershmekel) [inline tools/data_builder.py:184]: Please use a more standard time format such as:

```
>>> from datetime import datetime, timezone
>>> datetime.now(timezone.utc).isoformat()
'2015-01-27T05:57:31.399861+00:00'
```
Also I would use the key `updated_at` as it's less ambiguous.

Comment (ubershmekel) [reply tools/data_builder.py:184]: Also I would want all the files to use the same timestamp

**Comment by ubershmekel** [comment]
Normally I'd wait, but I can just make these fixes myself. brb.

**Comment by ubershmekel** [comment]
I have the changes tested locally, but I'm struggling a bit with the git-github complex. We just made reviews mandatory, so I can't just push to master to merge my changes on top of this PR, but I also can't just modify this PR.

**Comment by samgoldman** [comment]
Thanks for making these changes. I think in the future you'd want to checkout my fork and make the changes to the appropriate branch, which would update the PR, but I'm not 100% sure.

**Comment by ubershmekel** [comment]
@samgoldman thank you. That is indeed what I was supposed to do. Did you have to actively let me modify your branch on your repo? Or does github do that by default for PRs? Just kind of surprised me I could push to your repo.

**Comment by samgoldman** [comment]
> 
> 
> I figured the git nonsense out. I do have permission to push to someone else's branch when they open a PR. Either that or this person had the foresight to let me. Pushing the changes.

@ubershmekel yeah, there's a checkbox when creating a PR that allows anyone with write access to push to the fork: 
image

**Comment by ubershmekel** [comment]
How do you feel about the changes I added? Not too late if you disagree with them.

**Comment by samgoldman** [comment]
I definitely agree with the better formatting and the consistent timezone. I'd probably move the initialization of `updated_at` to the top of the file so it's easier to find. Also probably go with all caps since it's not supposed to change.

**Comment by samgoldman** [comment]
Also, I noticed someone added you as a reviewer (presumably because you are the original author of the file). If you want, I can create a CODE_OWNERS file that will automatically add reviewers for specific files.

**Pull request merged**